tools/testing/source_code_test.py

```diff
@@ -91,6 +91,11 @@ def test_no_private_tf_api():
         "tensorflow_addons/optimizers/moving_average.py",
         "tensorflow_addons/metrics/r_square.py",
         "tensorflow_addons/utils/test_utils.py",
+        "tensorflow_addons/losses/contrastive.py",
+        "tensorflow_addons/losses/focal_loss.py",
+        "tensorflow_addons/losses/lifted.py",
+        "tensorflow_addons/losses/quantiles.py",
+        "tensorflow_addons/losses/triplet.py",
```

Comment:
Friendly reminder that those rules are here to enforce the usage of the public TF API only, by contributors in the future, so that we can be forward compatible with tensorflow. The blacklist is here to keep track of the things we have not migrated yet, and so should not grow :) Maybe I should have written a clearer message in the comment above.